ggshield/iac/collection/iac_diff_scan_collection.py

```diff
@@ -4,10 +4,12 @@
     CollectionType,
     IaCScanCollection,
 )
-from ggshield.iac.iac_scan_models import IaCDiffScanEntities, IaCDiffScanResult
+from ggshield.iac.iac_scan_models import IaCDiffScanResult
 
 
 class IaCDiffScanCollection(IaCScanCollection):
+    result: Optional[IaCDiffScanResult]
```

Comment:
I know it is not part of the MR, but we don't need a init in those classes, it is only for the `type` attribute which could be a class attribute since you never change it.